client/js/components/statement/DpBoilerPlateModal.vue

```diff
@@ -123,7 +123,7 @@ export default {
     },
 
     insertBoilerPlate () {
-      this.$emit('insertBoilerPlate', this.textToBeAdded)
+      this.$emit('insert', this.textToBeAdded)
```

Comment:
When renaming the event, it should be changed in every occurrence. I find `insertBoilerPlate` also in some Vue templates.

Reply:
Yes, thanks, I forgot to commit and push these. Done in [3bd206d](https://github.com/demos-europe/demosplan-core/pull/969/commits/3bd206dbc5709e3adca7889557b6f4901619810e)